### ISSUE TITLE
Extra website showing in Matomo dashboard in addition to the tracked one

### ISSUE THREAD

**Issue by wrtz008** [matomo-org/tracker-proxy#88]
I am using Proxy Tracker with latest Matomo, here are the specs:

- Matomo version: 5.1.2
- MySQL version: 10.6.20-MariaDB-ubu2204
- PHP version: 8.3.14
- Ubuntu 22.04

on a specific domain used for tracking `matomo-server.com`.

I am using this `matomo-server.com` to track multiple websites: `tracked-domain-1.com` and `tracked-domain-2.com`, by adding the `matomo folder` in the sites and using also the Token for the UserTrackingAPI.

Each of the `tracked-domain-1.com` and `tracked-domain-2.com` have the JS code added that looks like this:

```
<!-- Matomo -->
<script>
  var _paq = window._paq = window._paq || [];
  /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
  _paq.push(['trackPageView']);
  _paq.push(['enableLinkTracking']);
  (function() {
    var u="//tracked-domain-2.com/matomo/";
    _paq.push(["setTrackerUrl", u+"matomo.php"]);
    _paq.push(["setSiteId", "hashed-value-here"]);
    var d=document, g=d.createElement("script"), s=d.getElementsByTagName("script")[0];
    g.type="text/javascript"; g.async=true; g.defer=true; g.src=u+"matomo.php"; s.parentNode.insertBefore(g,s);
})();
</script>
<!-- End Matomo Code -->
```

The `tracked-domain-1.com` is the first website I've setup on `matomo-server.com` and the primary domain.

The issue I have now is that inside Matomo's dashboard for `tracked-domain-2.com`, I am seeing clicks and URLs from `tracked-domain-1.com`:

![matomo proxy tracker](https://github.com/user-attachments/assets/40cf8d1c-2c2e-4420-b2b7-2bdbf8d4927a)

**Any idea what what's the issue and solution here?
Is there any way I can troubleshoot this to find what's causing this?**

It's confusing to see 2 sites in the same dashboard : \

Thank you!

**Comment by jkdn0921s** [comment]
I'm having the same problem here - glad it's not just me!

**Comment by arnvctslpsdoo** [comment]
Yep, same issue on my end as well :|

**Comment by navirados** [comment]
Same issue happening on my side too.

**Comment by siioiool** [comment]
I can confirm this issue on my Proxy Tracker setup as well.
I'd note that i can see the issue on the `tracked-domain-2.com` but not on `tracked-domain-1.com` Matomo dashboard.

**Comment by earthyowll** [comment]
Experiencing the same glitch. Looks like it's not isolated.

**Comment by algbthris** [comment]
I’m hitting this issue too. Any suggestions for a fix?

**Comment by wrtz008** [comment]
> I can confirm this issue on my Proxy Tracker setup as well. I'd note that i can see the issue on the `tracked-domain-2.com` but not on `tracked-domain-1.com` Matomo dashboard.

Yes same for me, seeing it only on the `tracked-domain-2.com` Matomo dashboard.

**Comment by wrtz008** [comment]
Update on this issue:
- i've deleted ```tracked-domain-2.com``` completely from Matomo dashboard
- then re-added it
- then re-added settings for Proxy Tracker
- created a new JS from Matomo
- used the new JS (that has the new Token for Proxy Tracker) and embedded it in the pages on ```tracked-domain-2.com```

and .. voila, it is not showing the ```tracked-domain-1.com``` stats inside the ```tracked-domain-2.com``` Matomo visitor log.

I've then run the crontab again and seems it is only showing visitors for ```tracked-domain-2.com```.
I'll keep checking in the next few days.
